src/libs/logos.js

```diff
@@ -23,15 +23,15 @@ import { isAnvil, isBaseline, isBioDataCatalyst, isDatastage, isElwazi, isFirecl
 import * as Utils from 'src/libs/utils'
 
 
-export const getAppName = (longName = false) => Utils.cond(
+export const getAppName = ({ longName = false, capitalInitial = false } = {}) => Utils.cond(
```

Comment:
I think it would be preferable to rename this argument to clarify that it only applies to RARE-X.

Reply:
Why is that necessary? An editor like IntelliJ makes it easy to find out the currently only use case in a snap, and I'd rather maintain the argument name reflect its functionality, which doesn't need to be specific to RARE-X.

Reply:
I find `capitalInitial` to be confused. I expected it to be whether the first word of the application (FireCloud, eLwazi, RARE-X" was capitalized, not "the". And I would expect the default to be `true` instead of `false`.

Where is this desired with `false`? I'm wondering if something creative can be done to avoid the need to introduce this variable.

Reply:
@kyuksel can you possibly use https://www.geeksforgeeks.org/lodash-_-lowerfirst-method/ for the special place(s) where you want the first letter to be lowercase?

Reply:
With this change, the function no longer works as advertised. If I call `getAppName({ capitalInitial: false })` in AnVIL, I would expect to get "the NGHRI AnVIL ...". But that's not the case. This parameter's only purpose is to special case the RARE-X label.